mongo/collection.go

```diff
@@ -724,8 +799,14 @@ func aggregate(a aggregateParams) (*Cursor, error) {
 	return cursor, replaceErrors(err)
 }
 
-// CountDocuments gets the number of documents matching the filter.
-// For a fast count of the total documents in a collection see EstimatedDocumentCount.
+// CountDocuments returns the number of documents in the collection. For a fast count of the documents in the
+// collection, see the  EstimatedDocumentCount method.
```

Comment:
nit: an extra space between the and EstimatedDocumentCount

Reply:
done.